packages/client/base/src/types/hosted/index.ts

```diff
@@ -43,8 +43,8 @@ export type CrossmintPayButtonProps = BaseButtonProps & {
 
 export type CheckoutProps = {
     experimental?: boolean;
```

Comment:
not related, but we should remove this lol

Reply:
yeah, but will do on a future release, I want to clean the experimental concept for the whole thing at once (sdk and main)

https://linear.app/crossmint/issue/PAY-2580/remove-the-experimental-flag